docs/DG_Diagrams/ExecFavCommand.puml

```diff
@@ -0,0 +1,73 @@
+@startuml
+box Logic
+participant ":Parser"
+participant ":ExecFavCommand"
+participant ":ExecFavParser"
+participant "p:Parser"
+end box
+
+box Model
+participant ":FavList" <<(C,#ADD1B2) Class>>
```

Comment:
![e1](https://user-images.githubusercontent.com/60377900/97543690-8b58aa80-1a03-11eb-9f68-6682b6ca3f7d.png)

Is the 'C' Logo supposed to be there?